src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java

```diff
@@ -57,6 +55,15 @@ public class OpenLoadFlowParameters extends AbstractExtension<LoadFlowParameters
      */
     public static final double SLACK_BUS_P_MAX_MISMATCH_DEFAULT_VALUE = 1.0;
 
+    /** Default value of the maximum active power mismatch in MW **/
+    public static final double MAX_ACTIVE_POWER_MISMATCH_DEFAULT_VALUE = Math.pow(10, -2);
+
+    /** Default value of the maximum reactive power mismatch in Mvar **/
+    public static final double MAX_REACTIVE_POWER_MISMATCH_DEFAULT_VALUE = Math.pow(10, -2);
```

Comment:
I find the default value a bit large as compared to other softwares , I was thinking something like 10-5 would make more sense

Reply:
Just for intel, here it is 10^-2 MVar for each equation (basically the balancing at each PQ node). So for instance 10^-5 means a precision of 100 Var at each node.